config/locales/en.yml

```diff
@@ -892,19 +917,20 @@ en:
       before_start_heading: Before you start
       before_start_paragraphs:
         - It should take about 15 minutes to answer all the questions.
-        - "You'll need information from your client (and any partner) about their:"
+        - "You'll need information about your client's (and their partner, if applicable.):"
       required_information_types:
-        - income, including any benefits they claim
-        - outgoings
-        - savings and investments
-        - assets, like property and vehicles
+        - "Income including any salary, benefits and financial support received"
+        - "Outgoings such as housing costs, childcare costs, maintenance and any criminal legal aid contribution payments your client may be making"
+        - savings, investments and other assets
```

Comment:
Grammar: Either all these bullet points should start with a capital, or none should.